secrethub_wrapper.go

```diff
@@ -58,4 +61,48 @@ func ResolveEnv(envVars map[string]string) map[string]string {
 	return resolvedEnv
 }*/
 
+// Exists checks if a secret exists at `path`.
+//export Exists
+func Exists(path *C.char, errMessage **C.char) C.bool {
+	client, err := secrethub.NewClient()
+	if err != nil {
+		*errMessage = C.CString(err.Error())
+		return C.bool(false)
```

Comment:
If anyone can find a better solution for this, so that we wouldn't actually have to return 'false' when an error is thrown, please, suggest it.
It is impossible to return a pointer to a boolean, since a compilation error gets thrown in this case: "Cannot take the address of (bool var)".

Reply:
It's fine like this. An exception will be thrown in all languages so the user will never see this value.